tests/e2e/specs/Modeler.spec.js

```diff
@@ -272,4 +272,51 @@ describe('Modeler', () => {
     cy.get('.paper-container').trigger('mouseup');
     cy.get('.ignore-pointer').should('have.length', 0);
   });
+
+  it.only('updates validation after undo/redo', () => {
+    cy.get('[data-test=validation-toggle]').click();
+    cy.get('[data-test=validation-list-toggle]').click();
+
+    cy.get('[data-test=validation-list]').children().should('have.length', 2);
```

Comment:
`should('have.length', 2);` 
Could this be extracted to a const? Same with other the magic numbers?

const `initialLoadValidationList`, `initialLoadValidationErrors`?